UI/PopulationPanel.cpp

```diff
@@ -70,7 +70,30 @@ void PopulationPanel::CompleteConstruction() {
     std::vector<std::pair<MeterType, MeterType>> meters;
 
     for (auto& meter_stat : m_meter_stats) {
-        meter_stat.second->InstallEventFilter(shared_from_this());
+        MeterType meter_type = meter_stat.first;
+
+        meter_stat.second->RightClickedSignal.connect([this, meter_type](const GG::Pt& pt) {
+            std::string meter_string = boost::lexical_cast<std::string>(meter_type);
+
+            auto popup = GG::Wnd::Create<CUIPopupMenu>(pt.x, pt.y);
+            auto pc = this->GetPopCenter();
+            if (meter_type == METER_POPULATION && pc) {
+                std::string species_name = pc->SpeciesName();
+                if (!species_name.empty()) {
+                    auto zoom_species_action = [species_name]() { ClientUI::GetClientUI()->ZoomToSpecies(species_name); };
+                    std::string species_label = boost::io::str(FlexibleFormat(UserString("ENC_LOOKUP")) %
+                                                                              UserString(species_name));
+                    popup->AddMenuItem(GG::MenuItem(species_label, false, false, zoom_species_action));
+                }
+            }
+
+            auto pedia_meter_type_action = [&meter_string]() { ClientUI::GetClientUI()->ZoomToMeterTypeArticle(meter_string); };
```

Comment:
still by reference here intentionally?

Reply:
No, that was an oversight.  Fixed via 17c6807c8d6a026690322d74e173afd3a2a359a3